frontend/src/_helpers/fake-backend.js

```diff
@@ -0,0 +1,57 @@
+export { fakeBackend };
+
+function fakeBackend() {
+  let users = [{ id: 1, username: 'admin', password: 'free5gc', firstName: 'Test', lastName: 'User' }];
```

Comment:
Note:
fakeBackend is a workaround.
@brianchennn will responsible for refactoring the current login mechanism.
Ref: https://github.com/free5gc/free5gc/issues/387

Reply:
@LaumiH
I am working to fix the [free5gc Issue #387](https://github.com/free5gc/free5gc/issues/387) about CSRF vulnerability. But it seems that your PR has changed a lot of files and directory. It's hard for us to merge and fix. Maybe you can wait us to fix the issue on **main** branch. Then, you can rebase our main branch of webconsole.

I hope you do not change too many files' name because it's hard to check all modifications by git tools.

Reply:
Hi @brianchennn, 

I think the mentioned CSRF vulnerability is fixed in the backend API code. I did not make breaking changes to the authentication mechanism in the backend (in fact, I use the fake backend to circumvent authentication altogether). The frontend only stores and uses the obtained authentication token from the (fake) backend.

My frontend changes are breaking. But as the issue is located in the backend, I don't see a big issue here, and it should be easy to rebase my changes in the backend with your bug fixes :) If you encounter any problems, I can also rebase my PR onto your changes in the main branch once they are published.

Cheers,
LaumiH

Reply:
> Note: fakeBackend is a workaround. @brianchennn will responsible for refactoring the current login mechanism. Ref: [free5gc/free5gc#387](https://github.com/free5gc/free5gc/issues/387)

The fake backend is a workaround, but I think it can be left in the project for easy testing and development. It will let through all backend calls (`realFetch`) except those that are intercepted and matched in the handleRoute switch case:
```JavaScript
      function handleRoute() {
        switch (true) {
          case url.endsWith('/login') && opts.method === 'POST':
            return authenticate();
          default:
            // pass through any requests not handled above
            return realFetch(url, opts)
              .then(response => resolve(response))
              .catch(error => reject(error));
        }
      }
```

Reply:
I added the real login mechanism to my frontend